CMakeLists.txt

```diff
@@ -126,20 +135,23 @@ endif()
 
 # Other files
 
-pkg_check_modules(BASHCOMP bash-completion)
-if (BASHCOMP_FOUND)
-    execute_process(COMMAND ${PKG_CONFIG_EXECUTABLE} --variable=completionsdir bash-completion OUTPUT_VARIABLE BASHCOMP_DIR OUTPUT_STRIP_TRAILING_WHITESPACE)
-    message("Bash completion directory: ${BASHCOMP_DIR}")
-    INSTALL(FILES createrepo_c.bash DESTINATION ${BASHCOMP_DIR} RENAME createrepo_c)
-    INSTALL(CODE "
-        execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink createrepo_c \$ENV{DESTDIR}${BASHCOMP_DIR}/mergerepo_c)
-        execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink createrepo_c \$ENV{DESTDIR}${BASHCOMP_DIR}/modifyrepo_c)
-        execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink createrepo_c \$ENV{DESTDIR}${BASHCOMP_DIR}/sqliterepo_c)
-        ")
-ELSE (BASHCOMP_FOUND)
-    INSTALL(FILES createrepo_c.bash DESTINATION "/etc/bash_completion.d")
-    message("Bash completion directory: /etc/bash_completion.d")
-endif (BASHCOMP_FOUND)
+OPTION(ENABLE_BASHCOMP "Install Bash autocompletions?" ON)
```

Comment:
Since we're being verbose for the other options, should this be named ```CREATEREPO_C_INSTALL_BASHCOMP``` to be more consistent?

Reply:
Yes. Ideally, option specific to the project should be prefixed by the project name.

Reply:
But this could be done in a different PR along with some other stylistic changes